packages/components/inputs/search-select-input/src/search-select-input.js

```diff
@@ -0,0 +1,151 @@
+import React, { useContext } from 'react';
+import PropTypes from 'prop-types';
+import { useIntl } from 'react-intl';
+import { ThemeContext } from '@emotion/core';
+import { SafeHTMLElement } from '@commercetools-uikit/utils';
+import { SearchIconDropdownIndicator } from '@commercetools-uikit/select-utils';
+import AsyncSelectInput from '@commercetools-uikit/async-select-input';
+import messages from '../../../../../src/components/internals/messages/select';
+import createSelectStyles from '../../../../../src/components/internals/create-select-styles';
+
+const useTheme = () => useContext(ThemeContext);
+
+const SearchSelectInput = (props) => {
+  const intl = useIntl();
+  const theme = useTheme();
+  const noOptionsMessageWithOutInputValue = props.initialSuggestionMessage
+    ? props.initialSuggestionMessage()
+    : intl.formatMessage(messages.noOptionsMessageWithoutInputValue);
+  const noOptionsMessageWithInputValue = props.noOptionsMessage
+    ? props.noOptionsMessage()
+    : intl.formatMessage(messages.noOptionsMessageWithInputValue);
+
+  return (
+    <AsyncSelectInput
+      horizontalConstraint={props.horizontalConstraint}
+      // react-select uses "id" (for the container) and "inputId" (for the input),
+      // but we use "id" (for the input) and "containerId" (for the container)
+      // instead.
+      // This makes it easier to less confusing to use with <label />s.
+      id={props.containerId}
+      inputId={props.id}
+      aria-label={props['aria-label']}
+      aria-labelledby={props['aria-labelledby']}
+      name={props.name}
+      placeholder={props.placeholder}
+      tabIndex={props.tabIndex}
+      iconLeft={props.iconLeft}
+      inputValue={props.inputValue}
+      value={props.value}
+      backspaceRemovesValue={props.backspaceRemovesValue}
+      hasWarning={props.hasWarning}
+      hasError={props.hasError}
+      isDisabled={props.isDisabled || props.isReadOnly}
+      isClearable={props.isClearable}
+      isOptionDisabled={props.isOptionDisabled}
+      isMulti={props.isMulti}
+      isSearchable={true}
```

Comment:
I'm thinking we could improve dev experience here by separating which props are we forwarding and which ones are we setting specifically for this SearchSelectInput. At the very least, we should sort them separately and add comments about them.

Perhaps we could either have a list of the `AsyncSelectInput` props to filter and forward directly, or just spread all remaining props, since they are already "filtered" and type-checked by the `AsyncSelectInput`.
This way, maintaining this component would be much easier when changes are made to the props of `AsyncSelectInput`, while also making the code of this component more "focused" on the value it brings by itself (adding a comment about the checking the implementation of the underlying AsyncSelectInput if needed)

Furthermore, for the README for this component, we could just document props that are new or have defaults, and refer to the documentation of the `AsyncSelectInput`, without duplicating everything,